internal/baseload/baseload.go

```diff
@@ -103,3 +103,11 @@ func (nl Load) ToResourceList() corev1.ResourceList {
 	nl.Apply(res)
 	return res
 }
+
+func (nl Load) CPULoad() resource.Quantity {
```

Comment:
I'm still very much in favor of adding these, but I'm surprised github is not complaining - we merged https://github.com/openshift-kni/numaresources-operator/pull/361 which changed the internal representation, so this PR actually needs a rebase

Reply:
Yep, on it.